### ISSUE TITLE
(Partially) Outdated documentation

### ISSUE THREAD

**Issue by rabauke** [OfficeDev/Open-XML-SDK#1025]
**Description**

Open-XML-SDK mainly refers to documentation for version 2.5 of the library, which is incompatible to the latest version. Many of the examples shown in the "How do I ..." section do not even compile with DocumentFormat.OpenXml 2.13.1. This makes it very difficult to start using this library.

**Expected**

Some up-to-date documentation including some elementary examples that illustrate how the various classes interact to create and to manipulate Open XML office documents.

**Comment by twsouthwick** [comment]
Can you give an example? The source code should compile just fine against the 2.13.1 build.

**Comment by rabauke** [comment]
@twsouthwick I tried to reproduce the observed incompatibilities and compilation issues on a different system. I was not able to reproduce them. Most likely, the reported issues indicate a problem that is related to the system where I observed the compilation problems (not to the Open-XML-SDK documentation). (Possibly caused by a _very_ old version of the SDK that is installed in parallel to the latest version, I need to investigate this in more detail.) Sorry for the noise.
